backend/src/appointment/routes/schedule.py

```diff
@@ -254,10 +254,10 @@ def request_schedule_availability_slot(
     db.commit()
 
     # Sending confirmation email to owner
-    background_tasks.add_task(send_confirmation_email, url=url, attendee=attendee, date=date, to=subscriber.email)
+    background_tasks.add_task(send_confirmation_email, url=url, attendee_name=attendee, date=date, to=subscriber.email)
```

Comment:
We need only the name here:

```suggestion
    background_tasks.add_task(send_confirmation_email, url=url, attendee_name=attendee.name, date=date, to=subscriber.email)
```